pkg/controller/node/node_controller.go

```diff
@@ -963,6 +963,10 @@ func (ctrl *Controller) updateCandidateMachines(pool *mcfgv1.MachineConfigPool,
 		if err := ctrl.setDesiredMachineConfigAnnotation(node.Name, targetConfig); err != nil {
 			return goerrs.Wrapf(err, "setting desired config for node %s", node.Name)
 		}
+		ctrl.logPool(pool, "Applying %s taint to %s", constants.NodeUpdateInProgressTaint.Key, node.Name)
```

Comment:
s/Applying/applying/

Reply:
I thought it is ok to have capital letters for log whereas it is recommended to not have error string starting with capitals.